routes/index.js

```diff
@@ -20,7 +22,11 @@ router.get('/', function(req, res, next) {
     }, function(lastBlock, callback) {
       var blocks = [];
 
-      var blockCount = 128;
+      nodeVersion = req.app.locals.nodeStatus.version;
```

Comment:
If this is queried every single time, then this value should be cached at start up time.